docs/pymc_aesara.ipynb

```diff
@@ -0,0 +1,1767 @@
+{
```

Comment:
<p>Let's not use JAX, that was used in the presentation because it's a new thing for devs, but might just confuse/distract users </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc/pull/5721/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>

Reply:
<p>Sure!</p>